src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,784 @@
+package seedu.duke;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+import java.io.IOException;
+import java.io.FileWriter;
+
+class DukeException extends Exception {
```

Comment:
Since the bot is no longer Duke should we rename the exception to reflect the name of the chatbot?